cellprofiler/modules/shrinkobjectstopoints.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+
+"""
+ShrinkObjectsToPoints
+======================
+
+**ShrinkObjectsToPoints** will transform a set of objects into a label image with single points
+representing each object. The location of each point corresponds to the centroid of the input objects.
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import skimage.measure
+
+
+class ShrinkObjectsToPoints(cellprofiler.module.ObjectProcessing):
+
+    module_name = "ShrinkObjectsToPoints"
+
+    category = "Advanced"
+
+    variable_revision_number = 1
+
+    def create_settings(self):
+
+        super(ShrinkObjectsToPoints, self).create_settings()
+
+        self.x_name.text = "Input objects"
+
+        self.x_name.doc = "Select the objects that you want to shrink to points."
+
+
+    def settings(self):
+
+        __settings__ = super(ShrinkObjectsToPoints, self).settings()
+
+        return __settings__
+
+
+    def visible_settings(self):
+
+        __settings__ = super(ShrinkObjectsToPoints, self).visible_settings()
+
+        return __settings__
+
+
+    def display(self, workspace, figure):
```

Comment:
You can use the super class' `display` method. Change these:

`workspace.display_data.x_data = input_objects.segmented`
`workspace.display_data.y_data = output_objects.segmented`